tools/build.sh

```diff
@@ -87,6 +87,10 @@ case "$1" in
     install)
         if [ -e "bin/theca" ]; then
             cp bin/theca $INSTALL_DIR/
+            if [ $? -ne 0 ]; then
+                p "theca could not be copied to $INSTALL_DIR"
+                err "please rerun the script with sudo if you do not have write permissions in $INSTALL_DIR"
```

Comment:
I can't say I love theca only using lowercase even at the beginning of sentences, but I guess it's a matter of taste so I thought it'd be easier to get this merged if I just follow the 'convention' :smiley_cat: